config/storage.yml

```diff
@@ -1,12 +1,8 @@
-test:
-  service: Disk
-  root: <%= Rails.root.join('tmp/storage') %>
-
-local:
+development:
   service: Disk
   root: <%= Rails.root.join('storage') %>
 
-minio:
+production:
```

Comment:
Buralarda acaba servisler için yapılandırmayı ayarlayıp production veya development ortamlarından hangisini kullanacağımızı belirlersek daha güzel olmaz mı?
Örnek:
```YML
local:	
  service: Disk
  root: <%= Rails.root.join('storage') %>

minio:
  service: S3
  access_key_id: <%= Rails.application.credentials.minio[:access_key] %>
  secret_access_key: <%= Rails.application.credentials.minio[:secret_key] %>
  endpoint: https://s3.omu.sh
  force_path_style: true
amazon:
  service: S3
  access_key_id: <%= Rails.application.credentials.minio[:access_key] %>
  secret_access_key: <%= Rails.application.credentials.minio[:secret_key] %>
  endpoint: https://s3.amazon.com
  force_path_style: true

developmet: :local
production: :minio

# veya

production:
  service: Mirror	
  primary: local	
  mirrors: [ local, s3 ]
```